core/src/sigverify.rs

```diff
@@ -51,6 +51,10 @@ fn verify_packet(packet: &Packet) -> u8 {
     let mut pubkey_start = pubkey_start as usize;
     let msg_start = msg_start as usize;
 
+    if sig_len == 0 {
```

Comment:
As mentioned in the pull request description, this part changes the runtime semantics. I'm assuming there will be no transaction with zero signature...